dbms/src/Flash/Planner/plans/PhysicalMockTableScan.cpp

```diff
@@ -115,6 +117,15 @@ void PhysicalMockTableScan::transformImpl(DAGPipeline & pipeline, Context & /*co
     pipeline.streams.insert(pipeline.streams.end(), mock_streams.begin(), mock_streams.end());
 }
 
+void PhysicalMockTableScan::transform(OperatorPipelineGroupBuilder & group_builder, Context & /*context*/, size_t /*concurrency*/)
+{
+    group_builder.addGroup(mock_streams.size());
```

Comment:
Should we check the `groups.size()`?
What will happen when `group.size()` > 0 in source Node?

Reply:
`groups.size() > 0` is ok here.
ref https://github.com/pingcap/tiflash/pull/6520/files#diff-05b2b05cec46a090ac7c1664829ffaca1c38b0459dbaf33f23ac15b6ffb982a6R40
A group represents a fine grained partition, although the group size must now be 1.
And `groups.size()` has been checked in `PipelineEvent::schedule()`

Reply:
PipelineGroups has been removed, and now `group_builder.init` is called here.
Checked will only be called once.